bundles/org.openhab.binding.melcloud/src/main/resources/OH-INF/thing/heatpumpDevice.xml

```diff
@@ -34,10 +34,10 @@
 				<label>Building ID</label>
 				<description>Building ID of the Heatpump device.</description>
 			</parameter>
-			<parameter name="pollingInterval" type="integer" required="true">
+			<parameter name="pollingInterval" type="integer" min="300" required="true">
```

Comment:
Is there any documentation that supports setting this as a minimum? It seems rather high as a minimum.

Reply:
As far as I know there isn't any official documentation on this.
Mitsubishi Electric announced limitations on X but didn't specify them:
https://twitter.com/ME_Europe/status/1755205001645801507?s=20
User reports on forum and issue indicate 360 works for them and most, including myself, also have success with 300.
To me it made sense to only allow values that we know will not trigger the "excessive traffic" message from the API.
I can lower it or remove it again if you want though..

Reply:
I would suggest to have it less strict. If users change this to a custom setting, and stumble accross the exception will know how to fix it. Creating a high limit will prevent any user from aadjust it if posslbe